addons/html_builder/static/src/core/building_blocks/basic_many2many.js

```diff
@@ -0,0 +1,84 @@
+import { Component, useRef, useState, onWillStart, onWillUpdateProps } from "@odoo/owl";
+import { useService, useAutofocus } from "@web/core/utils/hooks";
+import { debounce } from "@web/core/utils/timing";
+import { basicContainerBuilderComponentProps } from "./utils";
+import { DropdownItem } from "@web/core/dropdown/dropdown_item";
+import { Dropdown } from "@web/core/dropdown/dropdown";
+
+export class BasicMany2Many extends Component {
+    static template = "html_builder.BasicMany2Many";
+    static props = {
+        ...basicContainerBuilderComponentProps,
+        model: String,
+        fields: { type: Array, element: String, optional: true },
+        domain: { type: Array, optional: true },
+        limit: { type: Number, optional: true },
+        // createAction: { type: String, optional: true },
+        selection: { type: Array, element: Object },
+        setSelection: Function,
+        canCreate: { type: Boolean, optional: true },
+    };
+    static defaultProps = {
+        fields: [],
+        domain: [],
+        limit: 10,
+        canCreate: false,
+    };
+    static components = { Dropdown, DropdownItem };
+
+    setup() {
+        this.orm = useService("orm");
+        this.createInputRef = useRef("createInput");
+        this.state = useState({
+            selection: [],
+            searchResults: [],
+        });
+        this.onSearch = debounce(this.search.bind(this), 300);
+        // TODO focus on open dropdown, does not seem to work
+        useAutofocus();
```

Comment:
Because input is hide at the begining I think

Reply:
Moved to its own component